### ISSUE TITLE
performance problem

### ISSUE THREAD

**Issue by whph1991** [rockchip-linux/rknn-toolkit#196]
Thank you for kindly answering many questions in the meantime.

Thanks to this, we succeeded in driving our model with rockchip products.

However, there was a performance speed problem. Is there any way to improve this?

I used yolov5 as backbone and did quantization. And the default input image size is 1088x608 .

What I've tried before is to reduce the input image size. When reducing to 300x300, the execution speed increased, but the accuracy was decreased.

We are considering other ways to utilize tensorflow-lite.
pytorch -> onnx -> tensorflow -> tensorflow-lite -> rknn
Will this work?

Are there any other methods that you can recommend?
